### ISSUE TITLE
Add missing translation for uyghur language

### ISSUE THREAD

**Issue by teyyartap** [quran/quran.com-frontend-v2#459]
Please add some of Uyghur language translations to quran.com.
The following is one of the site this translation used: 
https://www.uyquran.com/ShowSure.aspx?ID=1
The ID is surah id and some of the words in translation has notes like this:
https://www.uyquran.com/ShowNote.aspx?id=2&sid=1
the id is note id (start from 1 in every surah) and sid is surah id.
This site used the third translation in this repository (file ug_3.db, SQLite database, also the DB.zip contains the Microsoft Access version of that database), which is translated by Enes. Alim.
https://github.com/teyyartap/translation
the first translation (ug_1.db) is translated by Muhammed Salih.
the second translation (ug_2.db) is translated by a commission based on the first translation.

**Comment by benomaire** [comment]
Thank you for reaching out!

If i understand correctly, you mention three translations:
1. ug_1.db by Muhammad Saleh (available on quran.com)
2. ug_2.db by a commission based on the first translation.
3. ug_3.db by Enes. Alim. (available on uyquran.com)

The first one is already available on quran.com. 
If you can provide any of the information below, that would be great!
1. who is the translator (basic biography)
2. what is the motivation for the translation
3. when it started/finished
4. what are the references used by the translator (Arabic Tafaseer, or other translations)
5. whether quran.com needs a license to use the translation
6. whether there is a pdf version of the translation

One last thing, if you want to translate the interface of quran.com to Uyghur, please register at https://translation.io/, and tell us your username there.
جزاكم الله خيرا

**Comment by teyyartap** [comment]
Yes, that's right. since the first one is already on quran.com. I will try to provide some information about the other two translations.

1. The second translation was translated by several people that arranged by Xinjiang Uyghur Autonomous Region Islamic Association. 
2. It is an improvement for the first translation and has published several different printout formations. 
3. It is finished in 2012. 
4. They referenced about 30 well-known Tafseer and other language translations.
5. As I know you don't need a license to use the translation.
6. There is a link to a PDF version: http://www.ewlad.biz/uyghurkitap/QedimiEserler/%D9%82%DB%87%D8%B1%D8%A6%D8%A7%D9%86%20%D9%83%DB%95%D8%B1%D9%89%D9%85%D9%86%D9%89%DA%AD%20%D8%A6%D8%A7%D8%AE%D9%89%D8%B1%D9%82%D9%89%20%D8%AA%DB%95%D8%B1%D8%AC%D9%89%D9%85%D9%89%D8%B3%D9%89.pdf

1. The third translation was translated by Enes Alim (also available at uyquran.com and quranterjime.com). He was well-known in his hometown and currently working at Sulaymaniyah Foundation as a researcher in Istanbul.
2. What differentiates his translation from others is in other translations that they used parenthesis in translation context to interpret a word or a situation, but he is used footnotes to explain anything necessary and the translation context goes fluent.
3. This translation was finished and published in 2016 in Istanbul, he said it takes almost 10 years to translate.
4. He also used a lot of well-known Tafseer and other translations too.
5. As I know it is no need for a license to use this translation.
6. I can't find a pdf for this translation.

Yes, I want to help translate the quran.com interface If i can. it is the same username teyyartap.
جزاكم الله خيرا

**Comment by benomaire** [comment]
You have been invited to translate the interface of quran.com on translation.io.
For the translations, thank you for the valuable details! I am not the one who will add the translations, but I asked what I expect would be useful for people considering them. I also informed quranenc.com.
جزاكم الله خيرا

**Comment by teyyartap** [comment]
Thank you so much!
جزاكم الله خيرا

**Comment by teyyartap** [comment]
The interface translation also finished but I can't see in the language options. Did I need to do something else? Or can i preview the translation in somewhere?
جزاكم الله خيرا
